deploy.sh

```diff
@@ -203,9 +216,12 @@ svn add . --force > /dev/null
 # Also suppress stdout here
 svn status | grep '^\!' | sed 's/! *//' | xargs -I% svn rm %@ > /dev/null
 
-# Copy tag locally to make this a single commit
-echo "➤ Copying tag..."
-svn cp "trunk" "tags/$VERSION"
+# Copy tag locally to make this a single commit when not
+# doing a trunk-only deployment.
+if [[ -n "$VERSION" && -z "$INPUT_TRUNK_ONLY" ]]; then
```

Comment:
Here is the core change -- we only add a tag if this is not a trunk-only release.